styleguide/_themes/global/scss/fonts.scss

```diff
@@ -49,6 +49,10 @@ p > a {
   @include standard-link;
 }
 
+li > a {
+  @include standard-link;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/5263371/29977153-02d1eb84-8f02-11e7-9afb-ced5b725eef0.png)

This seems to set the tab color to red as well.  

I don't think applying to `li > a` is a good idea.  We use `<li>` tags in way too many places.
I think a slightly safer idea would be to target li's inside specific content types where this is a problem. 

Unfortunately that requires undesirable nesting, but that's what the `zoolander/styleguide/_themes/global/scss/bootstrap_overrides.scss` file is for.